PluginsAndFeatures/azure-toolkit-for-intellij/rider/src/org/jetbrains/plugins/azure/functions/coreTools/FunctionsCoreToolsInfoProvider.kt

```diff
@@ -59,7 +61,28 @@ object FunctionsCoreToolsInfoProvider {
             }
         }
 
-        return FunctionsCoreToolsInfo(funcCoreToolsPath, coreToolsExecutablePath.path)
+        return FunctionsCoreToolsInfo(funcCoreToolsPath.path, coreToolsExecutablePath.path)
+    }
+
+    private fun patchCoreToolsPath(funcCoreToolsPath: File): File {
+
+        if (!SystemInfo.isWindows) return funcCoreToolsPath
```

Comment:
Maybe make a better validation here? We just need to check that application name is "func" and no extension. @maartenba, thoughts?

Reply:
This is essentially what is was before. A path check is done [here](https://github.com/JetBrains/azure-tools-for-intellij/pull/527/files/e372924c30015eb4cf5d36727c46f169a2918c5f#diff-c6d6d9a6e6734862278125aebfe3c5e5d197dd9508b025e2053bbff9a94f5f22R39) already